transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/LegacyFlgDestination.java

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.android.datatransport.cct;
+
+import static com.google.android.datatransport.cct.CctBackendFactory.mergeStrings;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.android.datatransport.runtime.Destination;
+import com.google.auto.value.AutoValue;
+import java.io.UnsupportedEncodingException;
+
+@AutoValue
```

Comment:
I don't think we need this to be auto-value, it only adds size and boilerplate, can you make it a plain class?